GitUpKit/Core/GCRepository.h

```diff
@@ -78,5 +78,6 @@ typedef NS_ENUM(NSUInteger, GCFileMode) {
 #if !TARGET_OS_IPHONE
 - (NSString*)pathForHookWithName:(NSString*)name;  // Returns nil if hook doesn't exist
 - (BOOL)runHookWithName:(NSString*)name arguments:(NSArray*)arguments standardInput:(NSString*)standardInput error:(NSError**)error;  // Silently ignores non-existing hooks
+- (BOOL)runHookWithName:(NSString*)name arguments:(NSArray*)arguments standardInput:(NSString*)standardInput standardOutput:(NSString**)standardOutput error:(NSError**)error;
```

Comment:
No need to add a new method. Just add the argument to the existing one and update the call sites.

This should be a separate commit BTW, the first one of the PR.